### ISSUE TITLE
QtCreator cannot find QtWebEngineView or QtWebView even after installing io.qt.qtwebengine.BaseApp/x86_64/6.2

### ISSUE THREAD

**Issue by herzenschein** [flathub/io.qt.QtCreator#62]
Attempting to add WebEngineWidgets to cmake renders:

CMakeLists.txt:15: error: Found package configuration file: /app/lib/cmake/Qt6/Qt6Config.cmake but it set Qt6_FOUND to FALSE so package "Qt6" is considered to be NOT FOUND.  Reason given by package: Failed to find Qt component "WebEngineWidgets". Expected Config file at "/app/lib/cmake/Qt6WebEngineWidgets/Qt6WebEngineWidgetsConfig.cmake" does NOT exist

As stated by the error, the file cannot be found in /app/lib/cmake. No Qt6WebEngineWidgets folder exists. You can test this by running `flatpak enter io.qt.QtCreator bash` and then `cd /app/lib/cmake`.

Attempting to use QtWebView in a QML file does show a window, but it's empty and doesn't work, producing the following error:

"No WebView plug-in found!"

Other general Widgets classes and Quick types seem to be found and work fine.

**Comment by 4c0n** [comment]
hi @herzenschein, thanks for reporting this. I've been working on the way this package is being built recently and noticed that the QtWebEngine module isn't built because it requires Python 2.7 which isn't available in the Sdk, the QtWebView module depends on QtWebEngine so that's probably the reason why that isn't working.
After refactoring the way the package is built, the problem will be fixed however it's taking me some time to get it all done.
The QtWebEngine BaseApp you mention in the title of this issue has no relation with this package, so I'm not surprised installing it doesn't help.

TL/DR; Will fix, but it'll take some time, thanks for your patience 🙂

**Comment by marbub** [comment]
QtWebEngine should now work.
